apps/web/app/(app)/mail/page.tsx

```diff
@@ -26,7 +26,7 @@ export default function Mail() {
 
   return (
     <>
-      {bannerVisible && typeof window !== "undefined" && (
+      {bannerVisible && (
```

Comment:
Thanks for flagging this. I believe there's a reason for `typeof window !== "undefined"`. There may have been a flash without it. I will take a look.